package.json

```diff
@@ -40,7 +40,7 @@
   "dependencies": {
     "@peculiar/asn1-schema": "^2.0.44",
     "@peculiar/json-schema": "^1.1.12",
-    "@typescript/lib-dom": "npm:@types/web",
+    "@types/web": "0.0.54",
```

Comment:
Please change the version to `^0.0.54`
Remove `package-locg.json` file and update `yarn.lock` file. This project uses `yarn`

```
npm un @types/web
npm i yarn ---global
yarn add @types/web
```